resources/locales/ar/cake.po

```diff
@@ -62,10 +58,15 @@ msgstr ""
 msgid "You are not authorized to access that location."
 msgstr "لا تملك صلاحية الدخول إلى هذه الصفحة"
 
-#: Error/ExceptionRenderer.php:222
+#: Error/Renderer/WebExceptionRenderer.php:367
 msgid "Not Found"
 msgstr "غير موجود"
 
+#: Error/error500.php:41
+#: Error/Renderer/WebExceptionRenderer.php:369
+msgid "An Internal Error Has Occurred."
```

Comment:
We should update the original message itself in core to use proper English and not capatilize every word :).

Reply:
The subject of this update is just cleanup.

Update of original message sounds as separate cake 5 task, because it affects both cakephp and app:

https://github.com/cakephp/cakephp/blame/bf3d59639b3cd7cf47e811be6d44c0dd6090a057/src/Error/Renderer/WebExceptionRenderer.php#L356
https://github.com/cakephp/app/blob/b8948f8aeb1d1d5991b9eb74d21c197ee7f26011/templates/Error/error500.php#L41

Reply:
Yes it's a separate task and needs to be done on a separate repo.

Reply:
Ok. Then lets accept this PR and then merge it into localized 5.x before chenge strings.